### DISCUSSION TITLE
Vercel Deploy Error: EPERM: operation not permitted, symlink

### DISCUSSION THREAD

**Discussion by NirajanMahara** [vercel/next.js#41676] [Help]
Vercel Deploy Error: EPERM: operation not permitted, symlink 'getExperience.func' -> 'D:\Builds\portfolio-2.0\.vercel\output\functions\api\getPageInfo.func'

**Comment by IvanTsxx** [comment]
You need to enable Windows Developer Mode so that Preconstruct can create symlinks - https://www.howtogeek.com/292914/what-is-developer-mode-in-windows-10/
